lms/djangoapps/bulk_email/admin.py

```diff
@@ -7,7 +7,13 @@
 from django.contrib import admin
 
 from lms.djangoapps.bulk_email.forms import CourseAuthorizationAdminForm, CourseEmailTemplateForm
-from lms.djangoapps.bulk_email.models import BulkEmailFlag, CourseAuthorization, CourseEmail, CourseEmailTemplate, Optout  # lint-amnesty, pylint: disable=line-too-long
+from lms.djangoapps.bulk_email.models import (  # lint-amnesty, pylint: disable=line-too-long
```

Comment:
We can remove this disable now.
```suggestion
from lms.djangoapps.bulk_email.models import (
```